### DISCUSSION TITLE
Problem with running isec='all' argument

### DISCUSSION THREAD

**Discussion by mkhalil8** [jonescompneurolab/hnn-core#721] [Q&A] [answered]
@rythorpe 
Hi Ryan,
I ran the simulation with the record_isec='all' argument, but encountered a couple of issues. Firstly, all the recordings in the cell_response.isec list returned zeros. Additionally, the simulation abruptly terminates.  if I do not specify the isec argument it runs well, even with vsec runs well. Below is the code snippet I used for simulation:
dpls_ampa2 = simulate_dipole(ampa2, tstop=tstop, n_trials=n_trials, record_isec='all')

**Comment by rythorpe** [comment]
@ntolley any idea what's going on here?

**Comment by ntolley** [comment]
@mkhalil8 is this on the master branch of `hnn-core` or the modified branch where you're introducing new cell types?

**Reply by rythorpe** [reply]
I just tested this on master and can't reproduce the issue. Perhaps you could give us more details about your specific situation @mkhalil8?

**Reply by mkhalil8** [reply]
@rythorpe @ntolley 
I had the problem with the master and the V0.4dev0 branch, I have the two versions on two separate Conda environments.  
I am running simulations in which I am running four simulations with four networks varying AMPA of the distal input with each simulation. I want to visualize the increase in the AMPA to the apical dendrites of L5 pyramidal. Plotting vsec did not demonstrate that, so I was experimenting with the isec. I was able to get the (record_vsec` = 'all' ) to work but not the isec

**Reply by rythorpe** [reply]
Do you get the same problem when you set `record_isec = 'all'` in [plot_simulate_evoked.py](https://github.com/jonescompneurolab/hnn-core/blob/master/examples/workflows/plot_simulate_evoked.py) example on master?

**Reply by mkhalil8** [reply]
I just did, adjusted the simulate dipole function:
```
with JoblibBackend(n_jobs=2):
    dpls = simulate_dipole(net, tstop=170., n_trials=2, record_isec='all')
```

The file contains two simulation with two trials each, the first simulation ran without problem and plotted the dipole and histogram. But the second simulation was killed(the one that uses net_sync, like what happened with me before.
I am using the master.

**Reply by jasmainak** [reply, marked as answer]
You may have a memory issue. Can you try with `n_jobs=1` or on a computer with more memory?

This reminds me of: https://github.com/jonescompneurolab/hnn-core/issues/684

**Reply by mkhalil8** [reply]
I am actually using Oscar, the super computer of Brown. I have been using 2 cores and 7GB memory. 
I tried with n_jobs= 1 but it was also killed. 
I initiated a new session with 20GB memory and it worked.
Thank you all
